onnxruntime/core/providers/cpu/object_detection/roialign.h

```diff
@@ -29,7 +29,6 @@ class RoiAlignBase {
       } else {
         ORT_THROW("Invalid mode of value ", mode, " specified. It should be either avg or max");
       }
-      mode_ = mode == "avg" ? RoiAlignMode::avg : RoiAlignMode::max;
```

Comment:
Don't we need this line? Not understanding how this can be removed...

Reply:
Lines 26 and 28 set `mode_` already (or error). This line is redundant.